### DISCUSSION TITLE
How to use automatic app event logging?

### DISCUSSION THREAD

**Discussion by MalikSamiAwan** [oddbit/flutter_facebook_app_events#94] [Q&A]
I am beginner developer, and i have build my first app, which is grocerry app and client demands for facebook app events, by searching alot i find this package usefull. I was able to configure(setup) this into my app. But i am not sure how to endable automatic app events (install, purchase) and where the result will be displayed.
For now, i have just write
 **### final facebookAppEvents = FacebookAppEvents();
  facebookAppEvents.setAdvertiserTracking(enabled: true);**
  these two lines in my main function of app, kindly help me out

**Comment by jpiabrantes** [comment]
I have the same question

**Reply by chiangandy** [reply]
I have same question too. after  I plugin sdk package and want to check any log information but get nothing in logging and network items. So confuse me how to make sure event message send successfully.

**Comment by DennisAlund** [comment]
Is this an iOS app or Android app? We just released fix for missing reportings in iOS (see PR #99 and issues #86 and #89)
